internal/roles/roles.go

```diff
@@ -82,34 +96,23 @@ func NewInstance(role string, parts ...string) *Instance {
 		case 2: // pool name
 			ins.Pool = v
 		case 3: // quota
-			n := mustParseInt(strconv.ParseInt(v, 10, 64))
+			n, err := humanize.ParseBytes(v)
+			if err != nil {
+				n = 0
```

Comment:
Should `NewInstance` return an error instead of setting the quota to 0? For example, if an invalid quota string is passed in, should we return a valid error message stating why it can't be used?

Reply:
Fixed